flow-extensions/src/test/kotlin/com/revolut/flowdata/extensions/CombineContentKtPairTest.kt

```diff
@@ -0,0 +1,87 @@
+package com.revolut.flowdata.extensions
+
+import com.revolut.data.model.Data
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+@ExperimentalCoroutinesApi
+class CombineContentKtPairTest {
+
+    private lateinit var aFlow: MutableSharedFlow<Data<String>>
+    private lateinit var bFlow: MutableSharedFlow<Data<String>>
+
+    private lateinit var testCombinedAB: Flow<String>
+
+    @BeforeEach
+    fun setup() {
+        aFlow = MutableSharedFlow()
+        bFlow = MutableSharedFlow()
+
+        testCombinedAB = combineContent(aFlow, bFlow)
+            .mapData { (a, b) -> a + b }
+            .extractContent()
+    }
+
+    @Test
+    fun `Combined values when both a and b has values`() = runFlowTest(testCombinedAB) {
+        aFlow.emit(Data("A"))
+        bFlow.emit(Data("B"))
+
+        Assertions.assertEquals("AB", expectMostRecentItem())
+        expectNoEvents()
+    }
+
+    @Test
+    fun `No combined values while both are loading`() = runFlowTest(testCombinedAB) {
+        aFlow.emit(Data(loading = true))
+        bFlow.emit(Data(loading = true))
+
+        expectNoEvents()
+    }
+
+    @Test
+    fun `No combined values when only a has value`() = runFlowTest(testCombinedAB) {
+        aFlow.emit(Data("A"))
+        bFlow.emit(Data())
+
+        expectNoEvents()
```

Comment:
@yatsinar actually, the tests (which I copied from CombineDataKtPairTest) uses in setup method extractContent, which filters content that is not null, so that's why here we expect noEvents. 
I'm gonna get rid of them extractContent and rewrite tests as expected. What do you think?